README.md

```diff
@@ -27,13 +27,12 @@ Find further information in the [Dynatrace community](https://community.dynatrac
 
 ### <a name="manual_installation"></a>Manual Installation
 
-* Download the [latest plugin](https://github.com/Dynatrace/Dynatrace-Maven-Plugin/releases) and extract it into the `lib` folder in your project
-* Import the Maven plugin into your local repository using the following command:
-`mvn install:install-file -DgroupId=dynaTrace -DartifactId=dtAutomation -Dversion=6.5.0 -Dpackaging=maven-plugin -Dfile=dtAutomation-6.5.0.jar`
+Plugin library is available in maven central repository (starting with version 7.0.0).
+
 * Define properties for the Dynatrace goals as shown in pom.xml from the sample package
-* Invoke your maven goals, e.g.: mvn dynaTrace:dtAutomation:6.2:startRecording
+* Invoke your maven goals, e.g.: mvn com.dynatrace.diagnostics.automation:dynatrace-maven-plugin:startRecording
```

Comment:
I know is too long, it was made to be similar to gradle. I will change it in both projects and publish once again.